package/package.json

```diff
@@ -20,7 +20,6 @@
         "build": "ts-node ./scripts/build.ts",
         "watch": "ts-node ./scripts/watch.ts",
         "typecheck": "tsc",
-        "test_release": "echo release > test/mode.txt && http-server -c-1 -p 8080 ./ -o index.html",
```

Comment:
While I haven't been using this, I think it still worked? Do you think we shouldn't need it?

Reply:
It's failing in runtime after you last change due to missing files. I manually copied from the dev folder and it worked.
But since we have the watch script you added, I don't think we need this.

Reply:
Watch script existed before my work, I just organized it differently, and used a different library. This existed alongside the watch script.

Reply:
Just tested it on my machine, it does work, but you need to build the app version.